### DISCUSSION TITLE
Old Device

### DISCUSSION THREAD

**Discussion by nivek1612** [BottlecapDave/HomeAssistant-OctopusEnergy#1105] [Q&A] [answered]
I had to remove and re-add my electric car to the Octopus app. That worked fine. But now I have two vehicles listed in the Integration as vehicles. How so remove the old one.

**Comment by BottlecapDave** [comment, marked as answer]
Hello. You should be able to go into the device representing the old car and delete/remove the device manually (the option is in the three dots)

**Comment by nivek1612** [comment]
How did I miss that 
Think I got it into my head it would delete the integration 
Thanks Dave
